src/material/form-field/_mdc-text-field-structure.scss

```diff
@@ -130,19 +134,15 @@
   .mdc-text-field--outlined {
     height: 56px;
     overflow: visible;
-    padding-left: 16px;
-    padding-right: 16px;
 
-    @include _supports-max {
```

Comment:
Heads-up that we inherited this `_supports-max` logic from MDC and reducing the specificity might be break-ey during the presubmit.